apps/api/src/app/environments-v1/usecases/output-renderers/in-app-output-renderer.usecase.ts

```diff
@@ -9,6 +9,35 @@ export class InAppOutputRendererUsecase {
   execute(renderCommand: RenderCommand): InAppRenderOutput {
     const { skip, disableOutputSanitization, ...outputControls } = renderCommand.controlValues ?? {};
 
-    return outputControls as any;
+    if (disableOutputSanitization) {
+      return outputControls as any;
+    } else {
+      const { subject, body, avatar, primaryAction, secondaryAction, data, redirect } = outputControls as any;
+
+      return {
```

Comment:
we could copy-paste or expose `sanitizeHtmlInObject` from the framework, but i don't think it's overkill to map it here once. the outputs won't change much over time, so it should be fine. plus, this approach is stricter, more transparent, and easier to debug than `sanitizeHtmlInObject`.